src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.ProjectSymbolReference.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CodeFixes.AddImport
+{
+    internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSyntax>
+    {
+        private class ProjectSymbolReference : SymbolReference
+        {
+            private readonly Project _project;
+
+            public ProjectSymbolReference(
+                AbstractAddImportCodeFixProvider<TSimpleNameSyntax> provider,
+                SymbolResult<INamespaceOrTypeSymbol> symbolResult,
+                Project project)
+                : base(provider, symbolResult)
+            {
+                _project = project;
+            }
+
+            protected override Glyph? GetGlyph(Document document)
+            {
+                return document.Project.Id == _project.Id
```

Comment:
Do we ever expect to have to generate a glyph for a document from another project?